chameleon/settings.py

```diff
@@ -244,6 +247,33 @@
 
 LOCALE_PATHS = (os.path.join(BASE_DIR, "locale"),)
 
+
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework.authentication.SessionAuthentication"
+    ],
+    # if not set explicitly, make admin only
+    "DEFAULT_PERMISSION_CLASSES": [
+        "rest_framework.permissions.IsAdminUser",
+    ],
+    # if not set explicitly, return json response.
+    "DEFAULT_RENDERER_CLASSES": [
+        "rest_framework.renderers.JSONRenderer",
+        "dynamic_rest.renderers.DynamicBrowsableAPIRenderer",
+    ],
+    "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.PageNumberPagination",
+    "PAGE_SIZE": 100,
+}
+
+DYNAMIC_REST = {
+    # DEBUG: enable/disable internal debugging
+    "DEBUG": False,
+    # ENABLE_BROWSABLE_API: enable/disable the browsable API.
+    # It can be useful to disable it in production.
```

Comment:
Should we disable it in production as recommended? What about setting these to be derived from the value of `DEBUG`?